rave-compiler/src/main/java/com/uber/rave/compiler/RaveProcessor.java

```diff
@@ -160,7 +163,8 @@ private ClassIR extractClassInfo(@NonNull TypeElement typeElement) {
             if (!executableElement.getModifiers().contains(Modifier.PUBLIC) && !samePackage) {
                 continue;
             }
-            MethodIR methodIR = new MethodIR(executableElement.getSimpleName().toString());
+            MethodIR methodIR = new MethodIR(executableElement.getSimpleName().toString(),
```

Comment:
I think there is an easier way to do this. Why not just add non null to the MethodIR if its not a primitive and is not nullable? Then we don't have to change all the logic.